newrelic/resource_newrelic_notifications_destination.go

```diff
@@ -69,11 +72,43 @@ func resourceNewRelicNotificationDestination() *schema.Resource {
 			"auth": {
 				Type:        schema.TypeMap,
 				Optional:    true,
-				ForceNew:    true,
 				Description: "A set of key-value pairs to represent a Notification destination auth.",
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				Sensitive:   true,
 			},
+			"active": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Indicates whether the destination is active.",
+			},
+			"account_id": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The account id of the destination.",
+			},
+
+			// Computed
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The status of the destination.",
+			},
+			"is_user_authenticated": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Indicates whether the user is authenticated with the destination.",
+			},
+			"last_sent": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The last time a notification was sent.",
+			},
+			"id": {
```

Comment:
I think we want to change the `id` attribute to be `destination_id` to be safe. Terraform has an internal `id` field for each resource and we set it by calling `d.SetId(<someID>)`. If an entity GUID exists, that's ideal and setting the resource ID would look would `d.SetId(entityGUID)`, so it's a little different than using `d.Set("id", ...)`.

Reply:
do you think we should do it for channels too?

Reply:
Yep, looks like you did that as well :)